README.md

```diff
@@ -36,13 +36,10 @@ We use GitHub issues in this repository to track features, bugs, and enhancement
 ## Credits and references
 
 Related projects
-  - https://github.com/cfpb/hmda-platform-ui
-  - https://github.com/cfpb/hmda-pub-ui
-  - https://github.com/cfpb/hmda-homepage
+  - https://github.com/cfpb/hmda-frontend
   - https://github.com/cfpb/hmda-help
   - https://github.com/cfpb/hmda-platform-tools
```

Comment:
platform-tools and documentation are also part of front-end. See [hmda-frontend](https://github.com/cfpb/hmda-frontend/blob/master/README.md).

Reply:
removed platform tools link